WordPress/Classes/ViewRelated/People/PersonViewController.swift

```diff
@@ -135,7 +135,7 @@ final class PersonViewController : UITableViewController {
         roleViewController.blog = blog
         roleViewController.selectedRole = person.role
         roleViewController.onChange = { newRole in
-            self.updateRole(newRole)
+            self.updateUserRole(newRole)
```

Comment:
Storing a strong reference to a parent to a child is not very recommendable, as it's almost sure the parent has a strong reference to the child as well (thus creating a retain cycle).

Since changing `self` to be `weak` here adds no risk to the code at hand, I'd consider doing so.